package.json

```diff
@@ -106,6 +106,45 @@
             "isSticky": true,
             "description": "Analyze a collection's schema."
           }
+        ],
+        "disambiguation": [
+          {
+            "category": "mongodb",
+            "description": "The user is asking for assistance or information related to MongoDB.",
+            "examples": [
+              "Explain this MongoDB query.",
+              "How do I write this MongoDB query in Python?",
+              "How do I use $lookup in MongoDB?",
+              "Can you help me debug a MongoDB query?",
+              "Rewrite this MongoDB query",
+              "How to convert this MongoDB query to Python?",
+              "How to convert this MongoDB query to Java?",
+              "How to convert this MongoDB query to Node.js?",
+              "How to convert this MongoDB query to Go?",
+              "How to convert this MongoDB query to php?",
```

Comment:
Do we want to include the rest of the supported languages? Currently `C#`, `Ruby`, and `Rust` are missing from the list of examples.

Reply:
Added below, good call out!